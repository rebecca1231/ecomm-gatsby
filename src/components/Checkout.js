import React from 'react'

import {loadStripe} from '@stripe/stripe-js'
import {Elements} from '@stripe/react-stripe-js'

import CheckoutForm from './CheckoutForm'
const STRIPE_PUBLISHABLE_KEY = "pk_test_51HH0LFHDlZ1F32WmvquMI1hPjncu5K1rCFHXscEYj6D58ZZFqyMKEQmKlRFXe6s7hbp7dJqzeAtmg01kZKEXp0H300VWeMqO7t"

const stripe = loadStripe(STRIPE_PUBLISHABLE_KEY)

export default ({}) => (
    <div>
        <Elements stripe={stripe}>
            <CheckoutForm />
        </Elements>
    </div>
)