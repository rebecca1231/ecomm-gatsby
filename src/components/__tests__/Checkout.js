import React from "react"
import renderer from "react-test-renderer"

import Checkout from "../Checkout"

describe("Checkout", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Checkout siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})