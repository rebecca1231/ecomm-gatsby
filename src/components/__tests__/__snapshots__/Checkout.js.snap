// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <div />
    <button
      disable={true}
    >
       Buy it 
    </button>
  </form>
</div>
`;
